lumicks/pylake/force_calibration/power_spectrum_calibration.py

```diff
@@ -155,12 +156,26 @@ def calculate_power_spectrum(data, sample_rate, fit_range=(1e2, 23e3), num_point
     num_points_per_block : int, optional
         The spectrum is first block averaged by this number of points per block.
         Default: 350.
+    compatibility_mode : bool
+        The original force calibration routine had a bug which resulted in a different downsampling
+        factor being used than specified, but then using the wrong downsampling factor in
+        subsequent computations (leading to a slightly different weighting of the data when
+        calibrating).
```

Comment:
* It would be good to include more info that you already have in the PR description. Specifically, which values does it affect and how bad is it.
* "The original force calibration" doesn't really describe what it is. Maybe specific that it's the calibration used by Bluelake 1.6.x and older.

Reply:
Changed it to:
```
The force calibration routine used in Bluelake 1.6.x and older had a bug which resulted in
a different downsampling factor being used than specified, but then using the wrong
downsampling factor in subsequent computations (leading to a slightly different weighting
of the data when calibrating). Enabling this flag computes a Power Spectrum which has the
same error when used for calibration. Note that this difference only resulted in small
deviations (starting from the third significant digit of most calibration properties for
a typical calibration spectrum). The only parameter that was largely effect was the
`chi_squared_per_deg`, which could be about 20% off.
```

Reply:
Looks good! One last request: a test?

Reply:
Certainly! Added! :)